infra/localstack.yaml

```diff
@@ -0,0 +1,18 @@
+version: '3.1'
+services:
+  localstack:
+    image: localstack/localstack
+    ports:
+      - "53:53"
+      - "443:443"
+      - "4510-4520:4510-4520"
+      - "4566-4620:4566-4620"
+      - "${PORT_WEB_UI-8080}:${PORT_WEB_UI-8080}"
+    environment:
+      - LOCALSTACK_API_KEY=$LOCALSTACK_API_KEY
```

Comment:
Suggestion (blocking): Can we inline the api key here? If it's a secret, please also remove the `.env` file that you accidentally included in the PR.

Reply:
I have idea how I ended up commiting that! Thanks for pointing it out